src/openforms/formio/components/custom.py

```diff
@@ -106,12 +107,18 @@ class NPFamilyMembers(BasePlugin):
 
     @staticmethod
     def _get_handler() -> Callable[[str], list[tuple[str, str]]]:
-        handlers = {
-            FamilyMembersDataAPIChoices.haal_centraal: get_np_children_haal_centraal,
-            FamilyMembersDataAPIChoices.stuf_bg: get_np_children_stuf_bg,
-        }
-        config = FamilyMembersTypeConfig.get_solo()
-        return handlers[config.data_api]
+        match FamilyMembersTypeConfig.get_solo().data_api:
+            case FamilyMembersDataAPIChoices.haal_centraal:
+                return get_np_children_haal_centraal
+            case FamilyMembersDataAPIChoices.stuf_bg:
+                return get_np_children_stuf_bg
+            case "":
+                logger.critical("No Family Members Data API configured")
+            case unknown:  # catch all
+                logger.critical(
+                    f'Invalid Family Members Data API "{unknown}" configured'
```

Comment:
don't use f-strings for log records, but use a proper interpolation format (and generally this implies old-style string formatting). That way, Sentry will group the errors by log message template instead of creating separate items for every one of them.